### PR TITLE
Handle some new date formats.

### PR DESCRIPTION
**Pull request by fiddlerwoaroof**

### CODE REVIEW

**Comment by chaitanyagupta** [comment]
Sorry, missed this out in the deluge of github notifications.

File: src/handler-defs.lisp [hunk 1]
```diff
@@ -237,3 +260,5 @@
     (tokens)
     ((repeater-time (? repeater-day-portion)))
   (get-anchor (dealias-and-disambiguate-time tokens)))
+
+nil
```
Comment (chaitanyagupta) [inline src/handler-defs.lisp]: Why add a `nil` at the end? Anyways, it seems irrelevant to the PR.

**Comment by chaitanyagupta** [comment]
I made a couple of fixes, added tests and merged this into master. Thanks for the patch!

**Comment by fiddlerwoaroof** [comment]
The nil at the end was to prevent the list of tokens from being printed out to the REPL when I ran the buffer via slime.  I forgot to remove it, but it's handy to have there for testing purposes.

**Comment by chaitanyagupta** [comment]
Ah. I redefined `DEFINE-HANDLER` to return the handler name instead of a list of handlers. That should fix this.
